Including C++ headers from extern "C" blocks
causes demangling of symbols and subsequent
incorrect resolution.

--- a/src/ffmpeg.imageio/ffmpeginput.cpp
+++ b/src/ffmpeg.imageio/ffmpeginput.cpp
@@ -2,8 +2,9 @@
 // SPDX-License-Identifier: BSD-3-Clause and Apache-2.0
 // https://github.com/AcademySoftwareFoundation/OpenImageIO
 
-extern "C" {  // ffmpeg is a C api
 #include <cerrno>
+
+extern "C" {  // ffmpeg is a C api
 #include <libavcodec/avcodec.h>
 #include <libavformat/avformat.h>
 #include <libswscale/swscale.h>
